proposals/20230511-roadmap-management.md

```diff
@@ -0,0 +1,93 @@
+# Falco Roadmap Management Proposal
+
+## Summary 
+
+This document proposes introducing a structured process for the management of the roadmap management and related changes in our development process to make effective the execution of our roadmap.
```

Comment:
This document proposes the introduction of a structured process for managing Falco's roadmap and implementing related changes in our development process. The goal is to ensure the efficient execution of our roadmap objectives.